src/sentry/features/__init__.py

```diff
@@ -14,6 +14,7 @@
 default_manager.add('organizations:callsigns', OrganizationFeature)  # NOQA
 default_manager.add('organizations:reports:prepare', OrganizationFeature)  # NOQA
 default_manager.add('organizations:reports:deliver', OrganizationFeature)  # NOQA
+default_manager.add('organizations:reports:calendar', OrganizationFeature)  # NOQA
```

Comment:
Kept this as-is for consistency right now, eventually we'll want to switch this to check the organization flags directly.